core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -253,12 +253,14 @@ class KafkaServer(val config: KafkaConfig, time: Time = SystemTime, threadNamePr
         ""
     }
 
+    val secureAclsEnabled = JaasUtils.isZkSecurityEnabled(System.getProperty(JaasUtils.JAVA_LOGIN_CONFIG_PARAM)) && config.zkEnableSecureAcls
```

Comment:
Sorry if this was already discussed, but shouldn't we be throwing a configuration error if someone sets `zkEnableSecureAcls` to `true` and `isZkSecurityEnabled` is false?